__tests__/data/indicative_input.json

```diff
@@ -431,5 +431,468 @@
+              "Model": "A5",
+              "Price": 13980,
+              "Mileage": 121015,
+              "DealerId": "972258cb-132f-62a9-bd31-e02f9f70fa7b",
+              "Derivative": "A5 Cabrio 2.0 TDI DPF  Cabrio"
+            },
+            {
+              "Vrm": "205536_3895",
+              "Make": "AUDI",
+              "Year": 2011,
+              "DName": "Autohaus Löbau GmbH",
+              "Model": "A5",
+              "Price": 12790,
+              "Mileage": 89236,
+              "DealerId": "9e697219-293e-63a0-884a-2d03f77bbbef",
+              "Derivative": "A5 2.0 TDI Sportback DPF  Coupe"
+            },
+            {
+              "Vrm": "116265",
+              "Make": "AUDI",
+              "Year": 2016,
+              "DName": "Elspass Autoland GmbH & Co. KG",
+              "Model": "A1",
+              "Price": 14690,
+              "Mileage": 42800,
+              "DealerId": "00c1bf3f-b19d-6198-a509-c7f2ff73c29a",
+              "Derivative": "A1 1.0 TFSI ultra  Limousine"
+            },
+            {
+              "Vrm": "010159",
+              "Make": "AUDI",
+              "Year": 2013,
+              "DName": "Hermann Meyer GmbH & Co. KG",
+              "Model": "A1",
+              "Price": 9950,
+              "Mileage": 93996,
+              "DealerId": "35d4fe7e-81a0-6e99-a575-6f62bffb8900",
+              "Derivative": "A1 1.4 TFSI Ambition Limousine"
+            },
+            {
+              "Vrm": "010178",
+              "Make": "AUDI",
+              "Year": 2016,
+              "DName": "Hermann Meyer GmbH & Co. KG",
+              "Model": "A1",
+              "Price": 12950,
+              "Mileage": 69454,
+              "DealerId": "35d4fe7e-81a0-6e99-a575-6f62bffb8900",
+              "Derivative": "A1 1.4 TFSI Sportback S tronic Attraction Limousine"
+            }
+          ],
+          "page": 0,
+          "merge": {
+            "nbHitsMax": 1000,
+            "nbHitslimit": 1000,
+            "nbSearchers": 2,
+            "nbHitsProcessed": 10,
+            "personalization": {
+              "impact": 100,
+              "enabled": true,
+              "profile": {
+                "time": 0,
+                "facets": {},
+                "taskID": 0
+              },
+              "percentile": 0,
+              "nbPersoScanned": 0,
+              "nbPersoSkipped": 0,
+              "nbPersoReranked": 0,
+              "nbPersoReturned": 0,
+              "nbPersoSelected": 0,
+              "nbRelevanceBuckets": 8
+            },
+            "lastHitToDisplay": 10,
+            "nbHitsNumberingEnd": 10
+          },
+          "nbHits": 13,
+          "nbPages": 2,
+          "queryID": "c5a176f8493236fcbfe69c3298d41659",
+          "indexUsed": "Germany_prod_stock_Finance_0",
+          "serverUsed": "d85-de-1.algolia.net",
+          "hitsPerPage": 10,
+          "parsedQuery": "audi a5",
+          "timeoutHits": false,
+          "appliedRules": [
```

Comment:
according to discussion, this should result in appliedRules.objectID="1539345377254" only right?

Reply:
No. I think it should be : `appliedRules.objectID = "1539345377254, 1548767806494"`